app/src/app/content-plugin/services/node-actions.service.ts

```diff
@@ -687,4 +697,10 @@ export class NodeActionsService {
       return moveStatus;
     }
   }
+
+  private static focusAfterClose(focusedElementSelector: string): void {
```

Comment:
does not need to be private static as it does not bring any improvement, besides `NodeActionsService.focusAfterClose` in all the method calls does not make sense, as those are inside the same class

Reply:
done